internal/xcuit/config.go

```diff
@@ -0,0 +1,136 @@
+package xcuit
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/saucelabs/saucectl/internal/config"
+	"gopkg.in/yaml.v2"
+)
+
+var supportedDeviceTypes = []string{"ANY", "PHONE", "TABLET"}
+
+// Project represents the xcuit project configuration.
+type Project struct {
+	config.TypeDef `yaml:",inline"`
+	ConfigFilePath string             `yaml:"-" json:"-"`
+	Sauce          config.SauceConfig `yaml:"sauce,omitempty" json:"sauce"`
+	Xcuit          Xcuit              `yaml:"xcuit,omitempty" json:"xcuit"`
```

Comment:
```suggestion
	Xcuit          Xcuit              `yaml:"xcuitest,omitempty" json:"xcuitest"`
```
We should stick to framework name which is `xcuitest`.

Reply:
I renamed this property.